### DISCUSSION TITLE
Transition between Grabbing and Backlight

### DISCUSSION THREAD

**Discussion by feuerwasser85** [hyperion-project/hyperion.ng#1485] [Ideas]
I use my LEDs for an indirect backlight too. My setup is a USB grabber and some WLED instances. This works just fine but switching between grabbing/not grabbing is not smooth. It would be nice to have a transitioning feature that allows me to set an amount of ms to switch between the backlighting WLED brings and USB grabbing. This can be switched disabling the led hardware, usb grabbing or the whole instance.

**Comment by YellowNest** [comment]
I asked the same question back in 2021. 

https://github.com/orgs/hyperion-project/discussions/1398

Hope the devs see this. It feels like it should be quite simple to implement
